services/src/main/java/edu/unc/lib/dl/cdr/services/processing/XMLImportJob.java

```diff
@@ -129,7 +129,7 @@ public void run() {
             log.info("Finished metadata import for {} objects in {}ms for user {}",
                     new Object[] {objectCount, System.currentTimeMillis() - startTime, username});
             sendCompletedEmail(updated, failed);
-        } catch (XMLStreamException e) {
+        } catch (FileNotFoundException | XMLStreamException e) {
```

Comment:
I don't think FileNotFoundException should be grouped together here (I know it was before). If the file isn't found, that probably indicates some kind of server error, like the temp file got deleted. It should also get logged at `log.error` level